common/util_test.go

```diff
@@ -896,79 +935,79 @@ func TestGenerateRandomString(t *testing.T) {
 		10: 10,
 	} {
 		t.Run(fmt.Sprintf("%d", input), func(t *testing.T) {
-			got := GenerateRandomString(input)
+			got := common.GenerateRandomString(input)
 			require.Len(t, got, wantSize)
 		})
 	}
 }
 
 func TestIsValidContext(t *testing.T) {
 	t.Run("background context", func(t *testing.T) {
-		require.NoError(t, IsValidContext(context.Background()))
+		require.NoError(t, common.IsValidContext(context.Background()))
 	})
 	t.Run("canceled context", func(t *testing.T) {
 		ctx, cancel := context.WithCancel(context.Background())
 		cancel()
-		got := IsValidContext(ctx)
+		got := common.IsValidContext(ctx)
 		require.Error(t, got)
 		require.ErrorIs(t, got, context.Canceled)
 	})
 	t.Run("deadline exceeded context", func(t *testing.T) {
 		ctx, _ := context.WithTimeout(context.Background(), -time.Second)
-		got := IsValidContext(ctx)
+		got := common.IsValidContext(ctx)
 		require.Error(t, got)
 		require.ErrorIs(t, got, context.DeadlineExceeded)
 	})
 	t.Run("context with deadline exceeded contextExpireThreshold", func(t *testing.T) {
 		ctx, _ := context.WithTimeout(context.Background(), contextExpireThreshold/2)
-		got := IsValidContext(ctx)
+		got := common.IsValidContext(ctx)
 		require.Error(t, got)
 		require.ErrorIs(t, got, context.DeadlineExceeded, "context.DeadlineExceeded should be returned, because context timeout is not later than now + contextExpireThreshold")
 	})
 	t.Run("valid context", func(t *testing.T) {
 		ctx, _ := context.WithTimeout(context.Background(), contextExpireThreshold*2)
-		require.NoError(t, IsValidContext(ctx), "nil should be returned, because context timeout is later than now + contextExpireThreshold")
+		require.NoError(t, common.IsValidContext(ctx), "nil should be returned, because context timeout is later than now + contextExpireThreshold")
 	})
 }
 
 func TestCreateChildContext(t *testing.T) {
 	t.Run("nil parent", func(t *testing.T) {
-		gotCtx, gotFunc := CreateChildContext(nil, 0)
+		gotCtx, gotFunc := common.CreateChildContext(nil, 0)
 		require.Nil(t, gotCtx)
-		require.Equal(t, funcName(emptyCancelFunc), funcName(gotFunc))
+		require.NotNil(t, gotFunc)
```

Comment:
arguably a better test would be to just call it - it's important that it's a no-op func that won't break something.

but yea, not-nil is I think sufficient here if you want to leave it.  it's clear enough.

Reply:
Reverted for now, but this is a pointless assertion.